### DISCUSSION TITLE
Tell players about the shop being full/out of stock when they try to sell/buy

### DISCUSSION THREAD

**Discussion by Andre601** [Ghost-chu/QuickShop-Reremake#1529] [Ideas]
When a player tries to sell or buy an item to/from a shop is QuickShop displays the sell/buy prompt, even if the shop in question has not enough space or items to keep buying or selling them.

Instead of the default sell/buy prompt should QuickShop post a message, telling about the shops current situation and that because of this, the player can't sell or buy to/from it.

**Comment by Ghost-chu** [comment]
Nice idea

**Reply by Andre601** [reply]
Has this been implemented already, or will it be?

**Reply by Ghost-chu** [reply]
still WIP

**Comment by Ghost-chu** [comment]
![image](https://user-images.githubusercontent.com/30802565/140599769-a4004015-d34b-42bd-88c3-d441e372efc1.png)

**Comment by Ghost-chu** [comment]
![image](https://user-images.githubusercontent.com/30802565/140599799-2bb0bc74-2a76-4e2c-a86a-be63b42454bb.png)
